src/components/Chat/Chat.tsx

```diff
@@ -794,6 +794,12 @@ function ChatLine(props) {{{
         );
       }
 
+      if (body.substr(0, 9) === '/senseis ') {
```

Comment:
If you want to learn about regexes, then there are lots of good sites out there. They look quite intimidating, but are really very simple. Eg https://regexone.com